recipes/distance/meta.yaml

```
@@ -0,0 +1,37 @@
+{% set name = "distance" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "60807584f5b6003f5c521aa73f39f51f631de3be5cccc5a1d67166fcbf0d4551"
```

Could you please remove the quotes from ``sha256``?
```suggestion
  sha256: 60807584f5b6003f5c521aa73f39f51f631de3be5cccc5a1d67166fcbf0d4551
```